Workbooks/Workloads/NGINX Insights/NGINX Insights.workbook

```diff
@@ -1080,7 +1080,7 @@
                   "name": "SuggestedAlerts",
                   "type": 1,
                   "isRequired": true,
-                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://raw.githubusercontent.com/acearun/managedsolutions/master/Templates/Alerts/Host/Memory/azuredeploy.json\"     } ]",
+                  "value": "[     {         \"AlertName\": \"host-high-cpu\",         \"AlertDescription\": \"Fires when high CPU usage is detected on the host\",         \"AlertThreshold\": 70,         \"AlertTemplate\": \"https://azmonsolutions.blob.core.windows.net/workloadinsights/Alerts/Host/Cpu/azuredeploy.json\"     },     {         \"AlertName\": \"host-used-memory-percentage\",         \"AlertDescription\": \"Fires when Used Memory percentage on the host exceeds a threshold\",         \"AlertThreshold\": 85,         \"AlertTemplate\": \"https://azmonsolutions.blob.core.windows.net/workloadinsights/Alerts/Host/Memory/azuredeploy.json\"     },     {         \"AlertName\": \"NGINX handled to accepted requests percentage\",         \"AlertDescription\": \"Fires if percentage of handled to accepted requests exceeds threshold\",         \"AlertThreshold\": 100,         \"AlertTemplate\": \"https://azmonsolutions.blob.core.windows.net/workloadinsights/Alerts/NGINX/nginx-handled-to-active-requests.json\"     } ]",

```

Comment:
You want 100% of accepts to be handled? Will this be noisy? Why not put in a bit of slack here? Also, the use of exceeds does not gel will 100%.